src/config.py

```diff
@@ -350,27 +340,47 @@ class PaymentConfig:
     @staticmethod
     def from_network(network: Network) -> PaymentConfig:
         """Initialize payment config for a given network."""
+        # pylint: disable=too-many-locals
         signing_key = os.getenv("PROPOSER_PK")
         if signing_key == "":
             signing_key = None
 
         docs_url = "https://www.notion.so/cownation/Solver-Payouts-3dfee64eb3d449ed8157a652cc817a8c"
 
-        payment_safe_address = Web3.to_checksum_address(
+        cow_token_address = Address("0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB")
+        payment_safe_address_cow = Web3.to_checksum_address(
+            os.environ.get(
+                "PAYOUTS_SAFE_ADDRESS_MAINNET",
+                "",
+            )
+        )
+        payment_safe_address_native = Web3.to_checksum_address(
             os.environ.get(
                 "PAYOUTS_SAFE_ADDRESS",
                 "",
             )
         )
 
+        # mainnet transaction nonces are increased by this modifier to allow for proposing
+        # multiple transactions on mainnet
+        nonce_modifier_dict = {
```

Comment:
With your code, you control the order but i guess the easiest and safest thing to do would be to just have

```
nonce_modifier_dict = {network: idx for idx, network in enumerate(Network)}
```